examples/spring-boot-draft/README.md

```diff
@@ -0,0 +1,107 @@
+SpringBoot Demo for using Jib
+
+Tips: this is just a draft, please contribute if you have good suggestion.
+
+### Quickstart
+
+1.Modify configuration:
+
+Modify `CUSTOM_REGISTRY_URL`, `YOUR_USER_NAME`, `YOUR_PASSWORD`, `PASSWORD_ENCRYPT_BY_MAVEN` as yourself, and change `credHelper` to match your platform .
```

Comment:
We'll most likely simplify the quickstart as we make edits towards finalization, but this level of detail is good for now :)